Documentation/format/index.md

```diff
@@ -94,11 +96,18 @@ After the labels, the number of indexed chunks is encoded, followed by a sequenc
 │ │                  │ └──────────────────────────┘     │ │
 │ ├──────────────────┼──────────────────────────────────┤ │
 │ │                  │ ┌──────────────────────────┐     │ │
-│ │                  │ │ c_i.mint <varint>        │     │ │
+│ │                  │ │ c_0.mint <varint>        │     │ │
+│ │                  │ ├──────────────────────────┤     │ │
+│ │                  │ │c_0.maxt - c_0.mint <uvarint>   │ │
+│ │                  │ ├──────────────────────────┤     │ │
+│ │                  │ │ ref(c_0.data) <uvarint>  │     │ │
+│ │      #chunks     │ └──────────────────────────┘     │ │
+│ │     <uvarint>    │ ┌──────────────────────────┐     │ │
+│ │                  │ │c_i.mint - c_i-1.maxt <uvarint> │ │
 │ │                  │ ├──────────────────────────┤     │ │
-│ │      #chunks     │ │ c_i.maxt <varint>        │     │ │
-│ │     <uvarint>    │ ├──────────────────────────┤ ... │ │
-│ │                  │ │ ref(c_i.data) <uvarint>  │     │ │
+│ │                  │ │ c_i.maxt - c_i.mint <uvarint>  │ │
+│ │                  │ ├──────────────────────────┤ ... │ │
+│ │                  │ │ ref(c_i.data) - ref0 <varint>  │ │
 │ │                  │ └──────────────────────────┘     │ │
```

Comment:
Can we make the columns all wider so it's nicely in boxes again ;(
I wanted them to be pretty so badly.